tests/rkt_volume_mount_kvm_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build kvm
```

Comment:
There's no difference between this and the nspawn one.

Can this just be one file of `rkt_volume_mount_non_fly` or such, and either be `!fly` or `coreos host kvm`?

We can split out stage specific files if/when we need to.